_contentTemplates/grid/built-in-dialogs.md

```diff
@@ -0,0 +1,13 @@
+#delete-confirmation
+The built-in Delete Confirmation Dialog requires minimum effort and code. Thus, the dialog itself does not provide customization options.
```

Comment:
This sounds like marketing content and can be skipped. Also, we rarely need to include excuses in the documentation.